server/src/migrations/05_denylist-table.ts

```diff
@@ -0,0 +1,28 @@
+"use strict"
+import { DataTypes } from "sequelize"
+import type { Migration } from "../database"
```

Comment:
this file should start with `06_` now, sorry! just make sure it's incremental with the existing migrations